clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2418,26 +2418,44 @@ device kernel, the attribute is not ignored and it is propagated to the kernel.
     [[intel::num_simd_work_items(N)]] void operator()() const {}
   };
 
-If the`` intel::reqd_work_group_size`` or ``cl::reqd_work_group_size``
```

Comment:
I think it would be more clear if we rearranged the changes to the docs. My recommendation is:

The docs for `reqd_work_group_size` is where most of these details should live. Those docs should be updated to explain what the three arguments are and that the argument which specifies the "work group size" differs depending on which spelling of the attribute is used. If possible, we should put an anchor in the docs for the work group size so that later links can direct users to that specific anchor.

The documentation that refers to `reqd_work_group_size` (like here) can then say "If the `reqd_work_group_size` attribute is specified on a declaration along with a `intel::num_simd_work_items` attribute, the requested work group size argument <link> must be evenly divisible by the argument specified in the `num_simd_work_items` attribute.

This way, we keep all the "we made bad decisions about the argument ordering" words are all in one place (instead of repeated) and can refer to the oddity from whatever other attributes have this sort of interaction with the work group size.